CONTRIBUTING.md

```diff
@@ -282,3 +285,18 @@ up-to-date master branch:
 This will create a new branch called 'miri' in your fork, and the output should
 include a link to create a rustc PR that will integrate those changes into the
 main repository.
+
+#### Authentication issues during push
+
+Unfortunately `josh` does not support ssh authentication. Going via https may cause
+josh to fail to authenticate with github. In that case you can additionally pass
+`--require-auth` to your `josh-proxy` invocation.
```

Comment:
Before we recommend this, could you maybe try adding this to your `~/.gitconfig`?
```
[url "git@github.com:"]
        pushInsteadOf = https://github.com/
```
I think that's what makes josh pushes work for me even though josh gets passed `--remote=https://github.com`.

Reply:
that did the trick, thanks!